library/node

```diff
@@ -3,114 +3,114 @@
 Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)
 GitRepo: https://github.com/nodejs/docker-node.git
 
-Tags: 15.2.0-alpine3.10, 15.2-alpine3.10, 15-alpine3.10, alpine3.10, current-alpine3.10
+Tags: 15.2.1-alpine3.10, 15.2-alpine3.10, 15-alpine3.10, alpine3.10, current-alpine3.10
```

Comment:
It seems the Alpine versions still have a bump to their tags. This might makes users think they are also security fixed but will still contain the old version.

Reply:
good catch! That's a bug in the script... I'll send a manual one

Reply:
Opened up #9098

Reply:
Opened up https://github.com/nodejs/docker-node/issues/1398 for the script bug